app/models/timeline_entry.rb

```diff
@@ -3,8 +3,10 @@
 class TimelineEntry < ApplicationRecord
   belongs_to :document
   belongs_to :user, optional: true
+
   has_one :retirement, foreign_key: :timeline_entries_id, dependent: :destroy, inverse_of: :timeline_entry
   has_one :removal, foreign_key: :timeline_entries_id, dependent: :destroy, inverse_of: :timeline_entry
+  has_one :internal_note, foreign_key: :timeline_entries_id, dependent: :destroy, inverse_of: :timeline_entry
```

Comment:
Should this be has_many?

Reply:
Nope i don't think so, as one timeline_entry is associated with one internal comment. E.g, you don't have one timeline_entry with multiple internal notes attached.

Reply:
Oh yeah right, I was thinking about something else. Nevermind, this looks good 👍